atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -224,8 +227,31 @@ def _render_mail(self, rebuild, success, auto_canceled, manual_canceled):
 
         url = self._get_logs_url()
 
+        repos = []
+        stored_data = self.workflow.exit_results.get(StoreMetadataInOSv3Plugin.key)
+        if not stored_data:
+            pass
```

Comment:
What's the purpose of these `pass` statements?

Reply:
Agreed.